src/Sylius/Bundle/SettingsBundle/Manager/SettingsManager.php

```diff
@@ -101,20 +102,29 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function loadSettings($namespace, $ignoreUnknown = true)
+    public function load($schemaAlias, $ignoreUnknown = true)
     {
-        if (isset($this->resolvedSettings[$namespace])) {
-            return $this->resolvedSettings[$namespace];
+        $schema = $this->schemaRegistry->getSchema($schemaAlias);
+
+        $resolver = $this->defaultResolver;
+
+        if ($this->resolverRegistry->has($schemaAlias)) {
+            $resolver = $this->resolverRegistry->get($schemaAlias);
         }
 
-        if ($this->cache->contains($namespace)) {
-            $parameters = $this->cache->fetch($namespace);
-        } else {
-            $parameters = $this->getParameters($namespace);
-            $this->cache->save($namespace, $parameters);
+        $settings = $resolver->resolve($schemaAlias);
```

Comment:
It ok, but we want to have mulitple Settings object, which will have the same schema. They will have different namespaces only.